### PR TITLE
test: try checking for maintainer check on PR comments

### PR DESCRIPTION
**Pull request by spring1843**
Fixes # 

**Description**

**How was this change tested?**

*

**Does this change impact docs?**
- [ ] Yes, PR includes docs updates
- [ ] Yes, issue opened: # 
- [x] No

**Release Note**

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/karpenter-comment.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: Karpentercomment
```
Comment (ellistarn) [inline .github/workflows/karpenter-comment.yaml:1]: wdyt about calling it something like this
```suggestion
name: manual-snapshot
```

File: .github/workflows/karpenter-comment.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+on:
+  issue_comment:
+    types: [created, edited, deleted]
```
Comment (ellistarn) [inline .github/workflows/karpenter-comment.yaml:4]: deleted?

File: .github/workflows/karpenter-comment.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+jobs:
+  approval-comment:
+    if: github.repository == 'aws/karpenter' && contains(github.event.comment.html_url, '/pull/') && contains(github.event.comment.body, '/karpenter snapshot')
```
Comment (ellistarn) [inline .github/workflows/karpenter-comment.yaml:8]: WDYT?
```suggestion
    if: contains(github.event.comment.html_url, '/pull/') && contains(github.event.comment.body, '/karpenter-snapshot')
```

**Review by ellistarn — APPROVED** [review]

File: .github/workflows/karpenter-comment.yaml (hunk 2 above)
Comment (spring1843) [reply .github/workflows/karpenter-comment.yaml:4]: removed

File: .github/workflows/karpenter-comment.yaml (hunk 3 above)
Comment (spring1843) [reply .github/workflows/karpenter-comment.yaml:8]: `/karpenter` in future can become bunch of different commands that all start with `/karpenter` I want the following strings to be parameters being passed to it

**Pull request merged**